packages/common/util.js

```diff
@@ -192,3 +193,15 @@ exports.lookupMimeType = (key) => mime.lookup(key) || null;
  * @kind function
  */
 exports.isOneOf = curry((collection, val) => collection.includes(val));
+
+/**
+ * Pass a value through a pipeline of functions and return the result
+ *
+ * @param {*} value - the value to be passed through the pipeline of functions
+ * @param {...Function} fns - the functions to be invoked
+ * @returns {*} the result of passing the value through the functions
+ *
+ * - If no functions are provided, the value is returned.
+ * - Functions should expect a single argument
+ */
+exports.thread = (value, ...fns) => flow(fns)(value);
```

Comment:
Why not just bring in flow directly where needed?  I can see it for dependency management/convenience, but in that case,  the name `thread` doesn't feel great.   I think we should use a name  that references flow, or at least doesn't bring to mind the other context 'threads/threading' can be used in.